cmd/ctr/commands/run/run_windows.go

```diff
@@ -91,6 +93,10 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 		}
 		if cwd := context.String("cwd"); cwd != "" {
 			opts = append(opts, oci.WithProcessCwd(cwd))
+		} else {
+			if snapshotter != "windows-lcow" {
+				opts = append(opts, oci.WithProcessCwd(`C:\`))
```

Comment:
OCI says this is required in both platforms. Why not have the generator make the defaults that can be overridden?

Reply:
This one, OK, I can live with. Will have a generated default. The other two, not.